consensus/types/src/chain_spec.rs

```diff
@@ -171,6 +171,15 @@ pub struct ChainSpec {
     pub subnets_per_node: u8,
     pub epochs_per_subnet_subscription: u64,
     pub attestation_subnet_extra_bits: u8,
+    // Shift networking configuration into configs
+    pub gossip_max_size: usize,
+    pub max_request_blocks: u64,
+    pub min_epochs_for_block_requests: usize,
+    pub max_chunk_size: usize,
+    pub ttfb_timeout: u64,
+    pub resp_timeout: u64,
+    pub message_domain_invalid_snappy: [u8; 4],
+    pub message_domain_valid_snappy: [u8; 4],
 
```

Comment:
Missing these https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L124C1-L129C34

Also, we would need to add the same values to the `Config` struct in this file. The `Config` struct is the rust representation of the `config.yml` file that we read for any given network.